src/util_misc.cpp

```diff
@@ -222,6 +222,22 @@ Util::skipBytes(int fd, size_t count)
   return totalSkipped;
 }
 
+ssize_t
+Util::readAllFromFile(const char *path, void *buf, size_t count)
```

Comment:
Why not change `readAllFromFile` to `readAll`?  Won't C++ allow this as two signatures for the same function name?